modernomad/settings.py

```diff
@@ -211,4 +212,21 @@
     '--nologcapture'
 ]
 
+
+class DisableMigrations(object):
+    def __contains__(self, item):
+        return True
+
+    def __getitem__(self, item):
+        return None
+
```

Comment:
cool. i see various Internet posts are using this approach from [django-tests-without-migrations](https://pypi.python.org/pypi/django-test-without-migrations), but not sure why it's a class?... is it because `MIGRATION_MODULES` expects a list? why not just return `[]`?